src/codegen.rs

```diff
@@ -378,8 +378,8 @@ fn codegen_struct(structure: &CheckedStruct, project: &Project) -> String {
     match structure.definition_type {
         DefinitionType::Class => {
             output.push_str(&format!(
-                "class {} : public RefCounted<{}> {{\n",
-                structure.name, structure.name
+                "class {} : public RefCounted<{}>, public Weakable<{}> {{\n",
```

Comment:
In the future, it would be nice to only enable `Weakable` for types that have `weak T`s pointing to them.

Reply:
That seems inconvenient to support across library boundaries